spec/rails_helper.rb

```diff
@@ -10,6 +10,7 @@
 require 'webmock/rspec'
 
 Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
+Dir[Rails.root.join('spec', 'lib', '**', '*.rb')].each { |f| require f }
```

Comment:
Instead of adding another lib folder (which can conflict later if you must write tests for a lib). What about using a `fixture` folder inside the existing `support` folder?

![image](https://github.com/nimblehq/ic-rails-phong-bliss/assets/77609814/d7323af0-ca63-4f9b-92b4-28706be3e52d)

Reply:
Great idea.